packages/@storefront/flux-capacitor/src/core/reducers/data/past-purchases.ts

```diff
@@ -43,17 +49,17 @@ export const DEFAULTS: State = <any>{
   query: '',
   sort: {
```

Comment:
I think you should add to the configuration adapter file https://github.com/groupby/storefront/blob/master/packages/%40storefront/flux-capacitor/src/core/adapters/configuration.ts#L33-L38
so that if someone wants to re-order their sort (or add additional sorts, like by price), they can do so and we will automatically set the types properly based on whether the given sort is in `SORT_FIELDS`

Reply:
I believe that update is described in the following ticket: https://groupby.atlassian.net/browse/SF-1240.

Can we leave that as a separate item of work?

Reply:
No. Being able to configure sorts and setting up sorts are the same implementation. For example, when sort was set up for search the configuration was done as the same ticket. It's one chunk of work in my opinion, and this feature is incomplete without it

Reply:
Updated implementation to allow implementations to specific past purchase sort options. Resolving.